src/ucp/core/ucp_context.c

```diff
@@ -125,6 +125,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "MD whose distance is queried when evaluating transport selection score",
    ucs_offsetof(ucp_config_t, selection_cmp), UCS_CONFIG_TYPE_STRING},
 
+  {"MEM_TYPE_REG_WHOLE_ALLOC", "y",
```

Comment:
can we shorten the name? and also MEMTYPE instead of MEM_TYPE to inline with other MEMTYPE env names. MEMTYPE_REG_ALLOC ?